ee/tabby-webserver/Cargo.toml

```diff
@@ -29,6 +29,7 @@ tabby-common = { path = "../../crates/tabby-common" }
 tarpc = { version = "0.33.0", features = ["serde-transport"] }
 thiserror.workspace = true
 tokio = { workspace = true, features = ["fs"] }
+tokio-cron-scheduler = "0.9.4"
```

Comment:
Unfortunately, `job_scheduler` has no async support, but db operation is async.

So import a new lib which inspired by `job_scheduler` but with async support